firstvoices/backend/resources/utils/import_export_widgets.py

```diff
@@ -80,7 +80,7 @@ class InvertedBooleanFieldWidget(Widget):
     """Import/export widget to return expected boolean value for audience related fields."""
 
     def clean(self, value, row=None, **kwargs):
-        curr_value = get_valid_boolean(value)
+        curr_value = get_valid_boolean_for_batch_import(value)
```

Comment:
I think this util function could be a method right in this widget, unless we are currently using it somewhere else